lib/foreman_maintain/utils/service.rb

```diff
@@ -11,4 +11,8 @@ def self.system_service(name, priority, options = {})
       Service::Systemd.new(name, priority, options)
     end
   end
+
+  def self.valid_sys_service?(service)
+    service.instance_of?(Service::RemoteDB) || service.instance_of?(Service::Systemd)
```

Comment:
Is it safe to list the possibilities? If somebody makes new subclass we would have to include in here.
`service.class < Service::Abstract` should work too.

Reply:
Still seems unresolved